test/e2e/app-dir/ppr-errors/ppr-errors.test.ts

```diff
@@ -1,8 +1,10 @@
 import { nextBuild } from 'next-test-utils'
+// In order for the global isNextStart to be set
+import 'e2e-utils'
 
 describe('ppr build errors', () => {
-  ;(process.env.TURBOPACK_DEV ? describe.skip : describe)(
-    'production mode',
+  ;(Boolean((global as any).isNextStart) ? describe : describe.skip)(
```

Comment:
Couldn't we move the whole test to `test/production/app-dir/ppr-errors` instead?

Reply:
Not sure what our policy here is? There are a lot of such tests in `e2e`
![Bildschirmfoto 2025-03-11 um 13 48 18](https://github.com/user-attachments/assets/d17751fd-aead-42b1-be4a-a8ee853de264)

Reply:
I think they should all be moved eventually. If a whole test suite does not test anything with `next dev` it should live in `test/production`. Otherwise we're just wasting CI cycles. https://github.com/vercel/next.js/blob/canary/contributing/core/testing.md#test-types-in-nextjs